aws/resource_aws_ecs_service_test.go

```diff
@@ -597,6 +615,86 @@ resource "aws_ecs_service" "mongo" {
 `, rInt, rInt)
 }
 
+func testAccAWSEcsServiceWithLaunchTypeFargate(rInt int) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {}
```

Comment:
All of our acceptance tests run in `us-west-2` by default and Fargate unfortunately isn't available there yet, so we'll need to pin this test to `us-east-1`, otherwise it fails:

```
=== RUN   TestAccAWSEcsServiceWithLaunchTypeFargate
--- FAIL: TestAccAWSEcsServiceWithLaunchTypeFargate (62.17s)
	testing.go:503: Step 0 error: Error applying: 1 error(s) occurred:

		* aws_ecs_service.main: 1 error(s) occurred:

		* aws_ecs_service.main: UnsupportedFeatureException: FARGATE is not supported in this region.
			status code: 400, request id: dc224bee-d9c1-11e7-a440-c32534bb3ba4 "tf-ecs-service-8420937161126328979"
```
We can do so by adding a `provider` block to the config here, e.g.

```hcl
provider "aws" {
  region = "us-east-1"
}
```
we do that for a couple of other services already for the same reason, like Lightsail.